blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -320,21 +320,26 @@ function get_font_family( $array, $configuration ) {
 		$variable = get_settings_array( $array, $configuration );
 		$slug     = preg_replace( '/var\(--wp--preset--font-family--(.*)\)/', '$1', $variable );
 		if ( ! isset( $this->fonts[ $slug ] ) ) {
-			$this->fonts[ $slug ] = $this->build_font_from_theme_data( $slug, $location, $configuration );
+			$this->fonts[ $slug ] = $this->build_font_from_theme_data( $slug, $configuration );
 		}
 		return $this->fonts[ $slug ];
 	}
 
-	function build_font_from_theme_data( $slug, $location, $configuration ) {
+	function build_font_from_theme_data( $slug, $configuration ) {
 		$new_font      = array();
-		$google_fonts  = $configuration['settings']['custom']['fontsToLoadFromGoogle'];
 		$font_families = $configuration['settings']['typography']['fontFamilies']['theme'];
 		foreach ( $font_families as $font_family ) {
 			if ( $font_family['slug'] === $slug ) {
 				$new_font['fontFamily'] = $font_family['fontFamily'];
 				$new_font['name']       = $font_family['name'];
 			}
 		}
+
+		// Get data about the font if we have it.
+		$google_fonts = $configuration['settings']['custom']['fontsToLoadFromGoogle'];
+		if ( empty( $google_fonts ) ) {
+			return false;
```

Comment:
This would mean the method could return FALSE instead of an array, whoich would set $this->fonts[ $slug ] to FALSE in get_font_family()... have you made sure that does not cause unexpected results?